admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -765,7 +771,7 @@ def _makeIndex(self, database, table):
         self._log.info('Generating index %r.%r', self.indexDb, metaTable)
 
         # try to delete existing table first
-        self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
+        #self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
```

Comment:
Can you add a comment why you commented-out this line so that we don't forget it in the future? Or if it's truly not needed better to remove it completely.